test/functional/p2p_leak.py

```diff
@@ -27,9 +26,10 @@
 )
 
 DISCOURAGEMENT_THRESHOLD = 100
+PING_MESSAGE_SIZE = 32
 
 
-class CLazyNode(P2PInterface):
+class CLazyPeer(P2PInterface):
```

Comment:
Since you're renaming these classes, you should drop the `C` prefix, which isn't the current code style.

Reply:
Done